mesonbuild/interpreterbase/decorators.py

```diff
@@ -290,10 +290,15 @@ class KwargInfo(T.Generic[_T]):
         different type. This is intended for cases such as the meson DSL using a
         string, but the implementation using an Enum. This should not do
         validation, just converstion.
+<<<<<<< HEAD
     :param deprecated_values: a dictionary mapping a value to the version of
         meson it was deprecated in.
-    :param since: a dictionary mapping a value to the version of meson it was
+    :param since_values: a dictionary mapping a value to the version of meson it was
         added in.
+=======
+    :param not_set_warning: A warning messsage that is logged if the kwarg is not
+        set by the user.
+>>>>>>> 3412c3776 (typing: Add not_set_warning to typed_kwargs)
```

Comment:
rebase conflicts?

Reply:
This is gone in the final diff... Is this from a specific commit?

Reply:
I will fix the commits once your Qt rewrite lands.